scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: hosts
+  gather_facts: False
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
+    register: _install_caasp_heat_templates
+    until: _install_caasp_heat_templates | success
+    retries: 5
+    delay: 2
+
+  - name: Download CaaSP image
+    always_run: True
+    get_url:
+      url: "{{ caasp_image_url }}"
+      dest: "{{ caasp_image_path }}"
+    register: _download_caasp_image
+    until: _download_caasp_image | success
+    retries: 5
+    delay: 2
+
+  - name: Upload CaaSP image to glance
+    always_run: True
+    shell: >
+      source ~ardana/service.osrc &&
+      openstack image create
+      --public
+      --disk-format qcow2
+      --container-format bare
+      --file "{{ caasp_image_path }}"
+      CaaSP-3
+
```

Comment:
would be good to delete the temporary download here after the image was created.